README.md

```diff
@@ -176,9 +200,9 @@ A device attribute MUST be one of these:
   </tr>
 </table>
 
-For example, a device with an ID of `686f6d6965` with a `base` and a `windsensor` node would send:
+For example, a device with an ID of `686f6d6965` with a `outdoor-probe` and a `windsensor` node would send:
 
-```
+```cpp
```

Comment:
The java without quotes is the prettiest IMHO. But the quotes might make the users think that "quoted strings" are of type `string` and that the rest are actually `boolean` and `numbers`, except it's not (everything is a `string`)